### ISSUE TITLE
Add local file strategy

### ISSUE THREAD

**Issue by nichtich** [gbv/login-server#29]
For testing purpose and small controlled lists of accounts. Requires rate limiting and encryption of passwords. I'd use [passport-local-htpasswd](https://www.npmjs.com/package/passport-local-htpasswd) for simplicity because `.htpasswd` files are well-known and because its made clear they are only for authentification (no additional user information).

**Comment by stefandesu** [comment]
I added local authentication via [passport-local](http://www.passportjs.org/packages/passport-local/). The users are written directly to `providers.json`, and [bcrypt](https://www.npmjs.com/package/bcryptjs) is used for hashing/salting the password. I also added a script to manage local providers and their users that allows creating/deleting providers/users (and hashes the user's password). Is this enough for this issue?
